### PR TITLE
Storage Release Version Update

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Netty and include OkHTTP client in your pom.xml.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.1.0-beta.1</version>
+  <version>1.0.0</version>
```
Comment (gapra-msft) [inline sdk/textanalytics/azure-ai-textanalytics/README.md]: Should I revert this?

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
 # Release History
 
-## 12.2.0-beta.2 (Unreleased)
+## 12.2.0 (2020-01-08)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob)
+
+- Release of 12.2.0-beta.1
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md]: Were the items below part of 12.2.0-beta.1 or part of the release of 12.2.0?

File: sdk/textanalytics/azure-ai-textanalytics/README.md (hunk 1 above)
Comment (alzimmermsft) [reply sdk/textanalytics/azure-ai-textanalytics/README.md]: @JimSuplizio

File: sdk/storage/azure-storage-blob/CHANGELOG.md (hunk 2 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/CHANGELOG.md]: The items below were added after 12.2.0-beta.1

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/CHANGELOG.md]: Do you think it would read better if I added it at the end?

Comment (weshaggard) [reply sdk/storage/azure-storage-blob/CHANGELOG.md]: I think you should just remove "- Release of 12.2.0-beta.1" as I'm not sure it is needed there given that the things under it were added after it. Also the version numbers are already in the headers so shouldn't be needed here.

File: sdk/storage/azure-storage-file-datalake/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 12.0.0-beta.9 (Unreleased)
+## 12.0.0-beta.9 (2020-01-08)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file-datalake_12.0.0-beta.9/sdk/storage/azure-storage-file-datalake/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file-datalake_12.0.0-beta.9/sdk/storage/azure-storage-file-datalake/src/samples/java/com/azure/storage/file/datalake)
+
+- Upgraded to version 12.2.0 of Azure Storage Blob.
```
Comment (weshaggard) [inline sdk/storage/azure-storage-file-datalake/CHANGELOG.md]: Delete "- Upgraded to version 12.2.0 of Azure Storage Blob."

File: sdk/storage/azure-storage-file-share/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 12.1.0-beta.2 (Unreleased)
+## 12.1.0 (2020-01-08)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file_12.1.0/sdk/storage/azure-storage-file-share/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file_12.1.0/sdk/storage/azure-storage-file-share/src/samples/java/com/azure/storage/file/share)
+
+- Release of 12.1.0-beta.1
```
Comment (weshaggard) [inline sdk/storage/azure-storage-file-share/CHANGELOG.md]: Delete "- Release of 12.1.0-beta.1"

File: sdk/storage/azure-storage-queue/CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 12.2.0-beta.2 (Unreleased)
+## 12.2.0 (2020-01-08)
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-queue_12.2.0/sdk/storage/azure-storage-file-queue/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-queue_12.2.0/sdk/storage/azure-storage-file-queue/src/samples/java/com/azure/storage/queue)
+
+- Release of 12.1.0-beta.1
```
Comment (weshaggard) [inline sdk/storage/azure-storage-queue/CHANGELOG.md]: Delete "- Release of 12.1.0-beta.1"

File: sdk/textanalytics/azure-ai-textanalytics/README.md (hunk 1 above)
Comment (JimSuplizio) [reply sdk/textanalytics/azure-ai-textanalytics/README.md]: I'd revert this file and let whomever owns azure-ai-textanalytics that this needs to be updated before they ship. Also, if we're shipping okhttp (I honestly don't know) then this version will change.

**Pull request merged**